components/input-group/InputFieldGroup.tsx

```diff
@@ -0,0 +1,90 @@
+import clsx from 'clsx';
+import React, { useId } from 'react';
+import Icon from '../icon/Icon';
+import { EIConTypes } from '../icon/icon.enum';
+import { TIconType } from '../icon/icon.interface';
+import { Label } from '../typography/Label';
+
+export interface IInputFieldGroupProps {
+  id: string;
+  text?: string;
+  placeholder: string;
+  as:
+    | 'text'
+    | 'password'
+    | 'email'
+    | 'number'
+    | 'tel'
+    | 'url'
+    | 'search'
+    | 'file'
+    | 'hidden';
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  state: 'default' | 'error' | 'success';
+  required?: boolean;
+  icon?: TIconType;
+  labelText: string;
+  errorMessage?: string;
```

Comment:
We could add following properties to the interface: 

- autoCorrect
- autoCapitalize
- spellCheck 

I would add default values of false to each one and make them optional.

Reply:
Thx for the hint. I didn't knnow these attributes before. I have implemented them now 😊